doc/website/blog/2023-12-12-hibernate-schema-loader.mdx

```diff
@@ -0,0 +1,597 @@
+---
+title: "Announcing Automatic Migrations for Hibernate Users"
+authors: dorav
+tags: [hibernate, schema, migration]
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+:::info TL;DR
+
+You can now import the desired database schema from your Hibernate project into Atlas,
+and use it to automatically plan migrations for you.
+
+[See an example](#demo-time)
+
+:::
+
+## Introduction
+
+Hibernate, is one of the most popular ORMs for Java, so much so that parts of it have evolved into
+the JPA standard and the jakarta APIs.
```

Comment:
Links supporting the claim?

Reply:
https://docs.jboss.org/hibernate/orm/6.4/introduction/html_single/Hibernate_Introduction.html#hibernate-and-jpa

I'll add it there, I am indeed missing a link to Hibernate